### ISSUE TITLE
Regional Model Limits combo box needs more functionality

### ISSUE THREAD

**Issue by norulz** [opengribs/XyGrib#152]
The purpose of the shading of the model area is not intuitive enough and users are not sure what it is for.

We can add a zoom to the shaded area (with some 10-15% extra margin) to make it clear where the area covered by the model is.

**Comment by dominiquehausser** [comment]
Yes good idea, as if you are zoomed on a narrow area, the shaded zone is out of visibility.
For me, using 1:1 for the full map is also OK, you see pretty well the zones.
